site/content/contribute/more-info/desktop/architecture/configuration.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Configuration"
+heading: "Configuration and Server Management"
+description: "Outlines the renderer processes that represent external Mattermost servers, and the modules that manage them."
+date: 2023-04-03T00:00:00-05:00
+weight: 1
+aliases:
+  - /contribute/desktop/architecture/configuration
+---
+
+### Config Module
+
+The **configuration** module in the **Common** module is responsible for facilitating reading from and writing to external configuration sources. It also consolidates, verifies and upgrades configuration where applicable.
```

Comment:
```suggestion
The **configuration** module in the **Common** module is responsible for facilitating reading from and writing to external configuration sources. It also consolidates, verifies, and upgrades configuration where applicable.
```

Reply:
@cwarnermm I thought our style guide was not to use Oxford commas? I personally prefer them though, so that change would be better for me 😛